talpid-core/src/tunnel/openvpn.rs

```diff
@@ -684,12 +685,13 @@ mod event_server {
             &self,
             request: Request<EventType>,
         ) -> std::result::Result<Response<()>, tonic::Status> {
-            log::info!("OpenVPN event {:?}", request);
-
             let request = request.into_inner();
+            log::debug!("OpenVPN event {:?}", request);
 
-            let event_type = openvpn_plugin::EventType::try_from(request.event)
-                .ok_or(tonic::Status::invalid_argument("Unknown event type"))?;
+            let event_type =
+                openvpn_plugin::EventType::try_from(request.event).map_err(|event: i32| {
+                    tonic::Status::invalid_argument(format!("Unknown event type: {}", event))
```

Comment:
Not sure if this has any practical difference. But with the new way `try_from` works we can get the invalid integer back, so we can add it to the error message.